includes/Renderer/Stories/Renderer.php

```diff
@@ -356,7 +356,7 @@ public function render_single_story_content(): void {
 		}
 
 		$this->assets->enqueue_script_module(
-			'web-stories-block-view',
+			'web-stories-embed-view-script-module',
 			$this->assets->get_base_url( 'assets/js/web-stories-block-view.js' )
```

Comment:
Come to think of it, with the change to the script module ID, is the src parameter even needed then? Won't it have been registered already with the block?

```suggestion
			'web-stories-embed-view-script-module'
```

Reply:
I register it here separately because the story might be embedded with a shortcode, not through the block.

Reply:
Wouldn't it still have been registered by WordPress? It would need to be enqueued here, but it wouldn't need to be registered, would it?